count_rate2counts should be dealing with the average, so no need for integers (#87)
sunpy/sunxspex: changelog/87.bugfix.rst

> Values are now left as floats when converting the model count rates (counts/second) to just counts since the model represents the average number of counts; therefore, there is not need for these to be integers.

Reviewer: ```suggestion
Removed cast to `int` when converting the model count rates (counts/second) to just counts since the model represents the average number of counts, there is not need for these to be integers.
```
